usaspending_api/api_docs/api_documentation/award_spending/recipient.md

```diff
@@ -1,3 +1,110 @@
-This is placeholder documentation for the /api/v2/award_spending/recipient
```

Comment:
Where's this coming from?  Did you just write it?  Why not make it a APIBlueprint compatible contract (and move it out of the docs directory)?

Reply:
I wrote this to be on the safe side, since this seems to be where all the "documentation for this page is located here" links to

Reply:
This PR also includes an APIBluprint for this same endpoint

Reply:
> This PR also includes an APIBluprint for this same endpoint

lol... didn't even notice.  We really only need one.  The docs stuff is what we're trying to move away from.  I'd nuke this any transplant the sample response stuff into the contract in a `Body` section.

Reply:
Then what will the link link to? Fixing that was the origonal point of this PR

Reply:
The contract?

Reply:
Ohh wait, we can set that in the view. My bad.